lib/raidenclient/RaidenClient.ts

```diff
@@ -157,7 +157,7 @@ class RaidenClient extends BaseClient {
       throw(errors.TOKEN_ADDRESS_NOT_FOUND);
     }
 
-    const tokenPaymentResponse = await this.tokenPayment(tokenAddress, destination, 1, rHash);
+    const tokenPaymentResponse = await this.tokenPayment(tokenAddress, destination, 1, `0x${rHash}`);
```

Comment:
I assume that we'll always want to prepend 0x when we make a token payment, so maybe we should put this in the `tokenPayment` method itself, line 381 I think. Also I just noticed we're using `Object.assign` there, it might be nice to create a `tokenPaymentRequest` type with a nullable `secret_hash` field so we can just do `payload.secret_hash =  `0x${secret_hash};`.

Reply:
Thanks for the feedback. Great idea - updated.